src/core/api/api.py

```diff
@@ -611,3 +611,43 @@ async def get_object_image(
     return StreamingResponse(
         _stream(gen), media_type="multipart/x-mixed-replace;boundary=frame"
     )
+
+
+@core_api.get(
+    "/projects/{project_id}/jobs/{job_id}/objects",
+    response_model=Dict[str, Any],
+)
+def get_objects_from_job(
+    project_id: int = Path(..., ge=1),
+    job_id: int = Path(..., ge=1),
+    start: int = Query(0, ge=0),
+    lenght: int = Query(10, ge=1),
+) -> Dict[str, Any]:
+    """Endpoint to get part of objects from a job.
+
+    Returns
+    -------
+    Dict[str, Any]
+        Containing total number of object in job and part of objects from
+        `start` to `start + lenght`.
```

Comment:
```suggestion
        `start` to `start + length`.
```

Reply:
This PR is based upon #72 and this is from there. It is all ready fixed in that PR.